ccpp/suites/suite_FV3_GSD_noah.xml

```diff
@@ -79,6 +79,7 @@
       <scheme>mp_thompson_post</scheme>
       <scheme>GFS_MP_generic_post</scheme>
       <scheme>cu_gf_driver_post</scheme>
+      <!-- <scheme>tracer_sanitizer</scheme> -->
```

Comment:
Is this  a place holder? tracer_sanitizer seems not used in any suite file?

Reply:
That's right, the "offending" `tracer_sanitizer` scheme is added in ccpp-physics for testing and debugging stability issues. It's not to be used ever in applications, because it's a dumb scheme that simply resets negative tracers to zero and doesn't care about water conservation. As such, it is commented out everywhere, but the commented lines for the two GSD suites illustrate how we used it in order to stabilize the model. It's similar to the Thompson MP tendency limiter, which simply puts a lid on the heating rate coming out of the Thompson scheme without adjusting any other quantity. I guess we will delete the `tracer_sanitizer` and Thompson MP tendency limiter once we figured out how to run the GSL physics in a stable way at reasonable time steps (it's always stable at small timesteps, but these can't be used in operations).